spec/ParseLiveQueryServer.spec.js

```diff
@@ -1183,6 +1308,139 @@ describe('ParseLiveQueryServer', function() {
         expect(isMatched).toBe(true);
         done();
       });
+
+    parseLiveQueryServer
+      ._matchesACL(acl, client, requestId)
+      .then(function(isMatched) {
+        expect(isMatched).toBe(true);
+        done();
+      });
+  });
+
+  describe('class level permissions', () => {
+    it('matches CLP when find is closed', done => {
```

Comment:
shouldn't this be 'does not match CLP if no read access'?

Reply:
the wording in the other tests is just saying the machesCLP method properly returns true or false i.e: the method works correctly. See https://github.com/parse-community/parse-server/pull/4387/files/095e38d56225f17397044d435c0ebe25a6f22e75#diff-7266dbfadf5cd4d0f9f8a9ec4536711aR1414